libraries/AP_HAL_ChibiOS/hwdef/AnyleafH7/defaults.parm

```diff
@@ -0,0 +1,25 @@
+# Onboard ELRS
+SERIAL5_PROTOCOL 23 # RCIN
+# RC_OPTIONS 512  # Suppress ELRS errors
+RC5_OPTION 154 # Set up arm switch to `ArmDisarm with AirMode`
+SERIAL5_BAUD 115
+RC2_REVERSED 1 # Prevent inverse pitch controls
+RSSI_TYPE 3 # Uart
+ARMING_RUDDER 0  # Disable rudder arming
+
+# Remove default serial protocols; set to None.
+SERIAL1_PROTOCOL -1
+SERIAL3_PROTOCOL -1
+SERIAL4_PROTOCOL -1
+
+
+# Onboard OSD
+OSD_TYPE 5 # MSP
+SERIAL2_PROTOCOL 42 # DisplayPort
+MSP_OPTIONS 4 # Betaflight fonts; without it, many symbols are missing.
+OSD_CELL_COUNT 0 # auto based on voltage at start
+
+
+# CAN
+CAN_P1_DRIVER 1 # "Enables use of CAN buses"
+GPS_AUTO_CONFIG 2 # Auto config for Serial and DroneCAN
```

Comment:
this should be the default unless you have a special DroneCAN GPS that needs config from the autopilot instead of locally